markdown/change-admin-password.md

```diff
@@ -0,0 +1,117 @@
+---
+slug: "change-admin-password"
+---
+
+## Changing the admin password
+
+Your new PMM Server will start up with a default set of credentials for the administrator account.
+When you first login via the UI, you will be prompted to change this to a new password in order to
+secure the account.
+
+When automating the deployment and/or management of your PMM Server, it is prefereable to use the
+available APIs instead of relying upon human interaction.
+
+The [authentication](#authentication) overview explains the different ways that you can
+programmatically access the API, using either [basic](#use-an-api-key-in-basic-auth), or
+[token-based](#authenticate) authentication. **Note:** for basic authentication, you can use the
+same approach for standard credentials if you haven't yet created a token.
+
+**Note** Examples using cURL will be shown with the `--netrc` argument, which allows
+credentials to be hidden from view in the processlist and shell history. Should you wish to
+use credentials in the command instead then substitute `--netrc` for `--basic --user '<user>:<password>'`
+
+Here is an example `.netrc`:
+```
+machine 127.0.0.1
+login admin
+password admin
+```
+
+### Check that the server is ready for you
+
+When the server starts up, there are a number of tasks that execute before the server and API
+are reliably available to the user. You can check whether the server is ready using the dedicated
+endpoint, [`/v1/readyz`](https://pmmdemo.percona.com/swagger#/Server/Readiness):
```

Comment:
> I think we also should mention that it's better to change admin password before configuring any pmm-agents, because it might bring to user lock.

There is already a note about this at the end

Reply:
I will move it up the page